groups/bsl/bslstl/bslstl_inplace.h

```diff
@@ -65,8 +78,81 @@ BSLS_KEYWORD_INLINE_CONSTEXPR in_place_t in_place = in_place_t();
 #else
 extern const in_place_t in_place;
 #endif
-    // Value of type 'in_place_t' used as an argument to functions that take an
-    // 'in_place_t' argument.
+// Value of type 'in_place_t' used as an argument to functions that take an
+// 'in_place_t' argument.
```

Comment:
Please revert; the docstring should be indented 4 spaces.
```suggestion
    // Value of type 'in_place_t' used as an argument to functions that take an
    // 'in_place_t' argument.
```